english/sanfran/sanfran.md

```diff
@@ -1,5 +1,5 @@
 Strolling down market street, friendly smiles to passerby.
 Cold wind rustles minty green leaves.
-Soft summer glows over the front sidewalks.
+Soft summer glows over the back sidewalks.
 This is a season to remember.
-I want to go for [a bike ride](bikeride/bikeride.md).
+Do I want to go for [a bike ride](bikeride/bikeride.md)? Drink some teas? Hmmmm, what about some [Ginger Peachy Keen](bikeride/ginger-peachy-keen/ginger-peachy-keen.md)?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.